.github/workflows/typecheck.yml

```diff
@@ -29,5 +29,8 @@ jobs:
       - name: Install web dependencies
         if: steps.cache.outputs.cache-hit != 'true'
         run: pnpm install --frozen-lockfile
+      - name: Generate config files
+        if: steps.cache.outputs.cache-hit == 'true'
```

Comment:
I think we can remove the conditional here.
(Which should also make this work 😅)
```suggestion
```

Reply:
I've removed it to see what happens 🫣